src/steps/ms-defender/finding/__tests__/converters.test.ts

```diff
@@ -0,0 +1,95 @@
+import {
+  createFindingEntity,
+  createFindingsCveRelationship,
+  createMachineFindingsRelationship,
+} from '../converters';
+import { Entity } from '@jupiterone/integration-sdk-core';
+import { Finding } from '../../../../types';
+const machine: Entity = {
+  _class: 'Machine',
+  _key: 'microsoft_defender_machine_id',
+  _type: 'microsoft_defender_machine',
+  displayName: 'name',
+};
+
+const finding: Entity = {
+  _class: 'Finding',
+  _key: 'microsoft_defender_finding_id',
+  _type: 'microsoft_defender_finding',
+  displayName: 'name',
+};
+
+const findingMock: Finding = {
+  id: '12345',
+  name: 'abc',
+  description: 'abc',
+  severity: 'high',
+  cvssV3: '123acdf',
+  exposedMachines: 'none',
+  publishedOn: '01/02/12',
+  updatedOn: '02/05/22',
+  publicExploit: true,
+  exploitVerified: false,
+  exploitInKit: false,
+  exploitTypes: [],
+  exploitUris: [],
+  _class: 'Finding',
+  _key: 'microsoft_defender_finding_id',
+  _type: 'microsoft_defender_finding',
+};
+
+const machineMock = {
+  _class: ['HostAgent'],
+  _type: 'microsoft_defender_machine',
+  _key: 'f8c2d6c26063babf52bc76979ef22f423387f3b2',
+  id: 'f8c2d6c26063babf52bc76979ef22f423387f3b2',
+  agentVersion: '10.8040.19041.1682',
+  defenderAvStatus: 'Unknown',
+  riskScore: 'High',
+  name: 'testmachine1',
+  computerDnsName: 'testmachine1',
+  displayName: 'testmachine1',
+  function: ['endpoint-protection', 'vulnerability-detection'],
+};
+
+describe('createFindingEntity', () => {
+  test('properties transferred', () => {
+    expect(createFindingEntity(findingMock, machineMock)).toBeTruthy();
+  });
+});
+
+describe('createFindingsCveRelationship', () => {
+  test('properties transferred for users', () => {
+    expect(createFindingsCveRelationship(machine)).toEqual({
+      _class: 'IS',
+      _key: 'microsoft_defender_machine_id|is|microsoft_defender_machine_id',
```

Comment:
This seems like a bad value, where both sides have the same value of `microsoft_defender_machine_id`, especially in the context of a finding -> cve relationship.